doc/release-notes/dash/release-notes-18.2.2.md

```diff
@@ -0,0 +1,171 @@
+
+Downgrading to a version older than v18.0.1 is not supported due to changes in
+the indexes database folder. If you need to use an older version, you must
+either reindex or re-sync the whole chain.
+
+Notable changes
+===============
+
+Testnet Breaking Changes
+------------------------
+
+A new testnet only LLMQ has been added. This LLMQ is of the type LLMQ_25_67; this LLMQ is only active on testnet.
+This LLMQ will not remove the LLMQ_100_67 from testnet; however that quorum (likely) will not form and will perform no role.
+See the [diff](https://github.com/dashpay/dash/pull/5225/files#diff-e70a38a3e8c2a63ca0494627301a5c7042141ad301193f78338d97cb1b300ff9R451-R469) for specific parameters of the LLMQ.
+
+This LLMQ will become active at the height of 847000. **This will be a breaking change and a hard fork for testnet**
+This LLMQ is not activated with the v19 hardfork; as such testnet will experience two hardforks. One at height 847000,
+and the other to be determined by the BIP9 hard fork process.
+
+Remote Procedure Call (RPC) Changes
+-----------------------------------
+
+### The new RPCs are:
+None
+
+### The removed RPCs are:
+None
+
+### Changes in existing RPCs introduced through bitcoin backports:
+None
+
+### Dash-specific changes in existing RPCs:
+None
+
+Please check `help <command>` for more detailed information on specific RPCs.
+
+Command-line options
+--------------------
+None
+
+Please check `Help -> Command-line options` in Qt wallet or `dashd --help` for
+more information.
+
+Backports from Bitcoin Core
+---------------------------
+None
+
+Other changes
+-------------
+#5247 is backported to improve debugging experience.
+
+v18.2.2 Change log
+==================
+
+See detailed [set of changes](https://github.com/dashpay/dash/compare/v18.2.1...dashpay:v18.2.2).
+
+Credits
+=======
+
+Thanks to everyone who directly contributed to this release:
+
+- Odysseas Gabrielides
+- UdjinM6
+
+As well as everyone that submitted issues, reviewed pull requests, helped debug the release candidates, and write DIPs that were implemented in this release.
+
+Older releases
+==============
+
+Dash was previously known as Darkcoin.
+
+Darkcoin tree 0.8.x was a fork of Litecoin tree 0.8, original name was XCoin
+which was first released on Jan/18/2014.
+
+Darkcoin tree 0.9.x was the open source implementation of masternodes based on
+the 0.8.x tree and was first released on Mar/13/2014.
+
+Darkcoin tree 0.10.x used to be the closed source implementation of Darksend
+which was released open source on Sep/25/2014.
+
+Dash Core tree 0.11.x was a fork of Bitcoin Core tree 0.9,
+Darkcoin was rebranded to Dash.
+
+Dash Core tree 0.12.0.x was a fork of Bitcoin Core tree 0.10.
+
+Dash Core tree 0.12.1.x was a fork of Bitcoin Core tree 0.12.
+
+These release are considered obsolete. Old release notes can be found here:
+
+- [v18.2.1](https://github.com/dashpay/dash/blob/master/doc/release-notes/dash/release-notes-18.2.2.md) released Jan/17/2023
```

Comment:
type 18.2.1.md -> 18.2.2md in file name

Reply:
@knst Good catch!

Reply:
this was in the archived release notes only, so not fixing here.